nemo_text_processing/text_normalization/taggers/measure.py

```diff
@@ -101,6 +105,25 @@ def __init__(self, cardinal: GraphFst, decimal: GraphFst, deterministic: bool =
             + pynutil.insert(" } ")
             + unit_singular
         )
-        final_graph = subgraph_decimal | subgraph_cardinal
+
+        subgraph_cardinal_dash = (
+            pynutil.insert("cardinal { integer: \"")
+            + cardinal_graph
+            + pynini.cross('-', '')
+            + pynutil.insert("\" } units: \"")
+            + pynini.closure(NEMO_NOT_SPACE, 1)
```

Comment:
what is this supposed to cover? 123-123? why? could we add to non deterministic?